extensions/gubbins/gubbins-client/src/gubbins/client/_generated/aio/operations/_patch.py

```diff
@@ -0,0 +1,50 @@
+# pylint: disable=line-too-long,useless-suppression
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""Customize generated code here.
+
+Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
+"""
+# bak = gubbins.client._generated.aio.operations._operations.WellKnownOperations
+# gubbins.client._generated.aio.operations._operations.WellKnownOperations = gubbins.client._generated.aio.operations.WellKnownOperations
+# # ------------------------------------
+
+from typing import List
+
+from ._operations import WellKnownOperations as _WellKnownOperations
+
+# from diracx.client._generated.aio.operations._patch import *
+# from diracx.client._generated.aio.operations._patch import __all__ as _patch_all
+
+
+class WellKnownOperations(_WellKnownOperations):
+    def my_method(self): ...
+
+
+__all__ = ["WellKnownOperations"]
+# __all__ += [x for x in _patch_all if x not in __all__]
```

Comment:
I don't think you meant to commit that

Reply:
Indeed, fixed (though this will will be autogenerated as part of https://github.com/DIRACGrid/diracx/issues/442).